proposals/drafts/disclosures.rst

```diff
@@ -0,0 +1,806 @@
+    date
+        The date the document was published in YYYY-MM-DD format
+        (partial dates are acceptable).
+    links
+        Links to 'available forms' of the document.  Each document can be available in
+        multiple forms such as PDF and HTML.  (For those familiar with DCAT this is the same
+        as the ``Distribution`` class.)
+        Has the following properties:
+
+        url
+            URL of the link.
+        media_type
+            The `media type <http://en.wikipedia.org/wiki/Internet_media_type>`_ of the link.
+
+sources
+    List of sources used in assembling this object.  Has the following properties:
+
+    url
+        URL of the resource.
+    note
+        **optional**
+        Description of what this source was used for.
+
+extras
+    Common to all Open Civic Data types, the value is a key-value store suitable for storing arbitrary information not covered elsewhere.
+
+Disclosure Type
+~~~~~~~~~~~~~~~
+
+identifier
+    An identifier that uniquely identifies the disclosure type.
+
+name
+    The canonical name of the disclosure type
+
+description
+    Description of the disclosure type
+
+action
+    The action performed by this disclosure type. Current values include:
+    
+    * registration  - registers a person or organization with a DisclosureAuthority
+    * report        - makes a periodic report to a DisclosureAuthority
+
+classification
+    The category of the disclosure type. Current values include:
+        
+    * lobbying      - Disclosures related to lobbying
+    * contributions - Disclosures related to political contributions
+
+amends_type
+    The identifier of the disclosure type that this disclosure type is able to amend. Can be the same as identifier, where future submissions supercede past submissions.
+
+amendment
+    **optional**
+    A boolean that is true if this is a registration type that is reserved for amending other registration types
+
+DisclosedEvent
+--------------
+The basis for the DisclosedEvent is the Open Civic Data ``Event`` type, as described in `OCDEP 4: Events <http://opencivicdata.readthedocs.org/en/latest/proposals/0004.html>`_. Constraints on field values specified below
+
+identifier
+    An identifier that uniquely identifies the event.
+
+classification
+    As defined in the ``Event`` type, where values are extended to include:
+
+    * lobbying
+    * contribution
+
+participants
+    Participants associated with the event. 
+
+    note
+        As defined on the ``Event`` type, where values identifies the role of the participant. Choices include the following, and may be updated:
+          * lobbyist      - (lobbying) person who actually did the lobbying
+          * lobbied       - (lobbying) organizations and/or individuals that were lobbied
+          * regarding     - (lobbying) bills, regulations or other matters that can be identified
```

Comment:
actually, yes it is.  it's role would be "regarding" (line 233), and i'm looking ahead to where we'd add regulations to that as well